src/app/containers/ConsentBanner/Banner/index.canonical.jsx

```diff
@@ -64,6 +72,7 @@ const CanonicalConsentBannerContainer = ({ type, onReject, onAccept }) => {
         )}
         script={script}
         service={service}
+        focusRef={consentBannerRef}
```

Comment:
Is there a reason we didn't use use `React.forwardRef()` on these Psammead components?

Reply:
Is that a bit less explicit? We wouldn't have any indication you can forward a ref to the component unless you read the docs? Or is this a common pattern?

Reply:
@simonsinclair We use `forwardRef` within the psammead-consent-banner component to forward it on to a DOM element, in this case the `h2`:

https://github.com/bbc/psammead/blob/64557df1b4aeba6fa646892d38e4df91a2fd722f/packages/components/psammead-consent-banner/src/index.jsx#L91-L98